CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.4.0] - 2022-11-11
+
+### Fixed
+
+- Fix backing format (#171)
+  - The PR #171 fixed a bug that the image is not created by `qemu-img create`.
```

Comment:
Please describe the supported qemu version.

Reply:
Add qemu version supported at fd0d0db0475874a1cd2dbe8d3b3c3e02029d87a3